libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -99,7 +99,7 @@ namespace SohImGui {
         "gCCMinimapPrim","gCCRupeePrim","gCCKeysPrim"        
     };
 
-    const char* filters[3] = {
+    std::vector<std::string> filters = {
```

Comment:
you can just use an std::array here and have access to the same operators. and it can stay a `const char*`

Reply:
Since its  only used in this function I would just make it an in function static array

Reply:
> Since its only used in this function I would just make it an in function static array

We did that since we reworked the entire function for the randomizer and where the enhancement combobox  were indeed completey screwed over from the recent merge and where we need to use lots of comboboxes

Reply:
To avoid unnecessary RAM storage and runtime boot initialization time, use `const char* const filters[] = ...`.